backend/core/src/shared/utils/format-local-date.ts

```diff
@@ -0,0 +1,20 @@
+import dayjs from "dayjs"
+import utc from "dayjs/plugin/utc"
+import tz from "dayjs/plugin/timezone"
+import advanced from "dayjs/plugin/advancedFormat"
+import { isEmpty } from "./is-empty"
+dayjs.extend(utc)
+dayjs.extend(tz)
+dayjs.extend(advanced)
+
+export const formatLocalDate = (
+  rawDate: string | Date,
+  format: string,
+  timeZone?: string
+): string => {
+  if (!isEmpty(rawDate)) {
+    const utcDate = dayjs.utc(rawDate)
+    if (!isEmpty(timeZone)) return utcDate.tz(timeZone.replace("-", "/")).format(format)
+    else return utcDate.format(format)
```

Comment:
You don't need to change this, but the "else" is unnecessary in this spot since  all code that fails the if statement we want to return what's in this else